dali/test/python/test_pipeline.py

```diff
@@ -1584,3 +1587,57 @@ def calc_avg_max(val):
         else:
             assert(calc_avg_max(v["real_memory_size"]) == v["max_real_memory_size"])
             assert(calc_avg_max(v["reserved_memory_size"]) == v["max_reserved_memory_size"])
+
+def trigger_output_dtype_deprecated_warning():
+    batch_size = 10
+    shape=(120, 60, 3)
+    pipe = Pipeline(batch_size=batch_size, num_threads=4, device_id=0)
+    data = RandomDataIterator(batch_size, shape=shape, dtype=np.uint8)
+    with pipe:
+        input = fn.external_source(data, layout="HWC")
+        cmn = fn.crop_mirror_normalize(input, device="cpu",
+                                                output_dtype=types.FLOAT,
+                                                output_layout="HWC",
+                                                crop=(32, 32),
+                                                image_type=types.RGB,
+                                                mean=[128., 128., 128.],
+                                                std=[1., 1., 1.])
+        pipe.set_outputs(cmn)
+    pipe.build()
+
+    result, = pipe.run()
+    assert result.as_array().dtype == np.float32
+
+
+def test_output_dtype_deprecation():
+    with warnings.catch_warnings(record=True) as w:
+        # Cause all warnings to always be triggered.
+        warnings.simplefilter("always")
+        # Trigger a warning.
+        trigger_output_dtype_deprecated_warning()
+        # Verify DeprecationWarning
+        assert len(w) == 1
+        assert issubclass(w[-1].category, DeprecationWarning)
+        assert ("Argument name 'output_dtype' for operator CropMirrorNormalize is deprecated. " +
+                "Use 'dtype' instead.") == str(w[-1].message)
+
+@raises(TypeError)
+def test_output_dtype_both_error():
+    batch_size = 10
+    shape=(120, 60, 3)
+    pipe = Pipeline(batch_size=batch_size, num_threads=4, device_id=0)
+    data = RandomDataIterator(batch_size, shape=shape, dtype=np.uint8)
+    with pipe:
+        input = fn.external_source(data, layout="HWC")
+        cmn = fn.crop_mirror_normalize(input, device="cpu",
+                                                output_dtype=types.FLOAT,
+                                            dtype=types.FLOAT,
+                                                output_layout="HWC",
+                                                crop=(32, 32),
+                                                image_type=types.RGB,
+                                                mean=[128., 128., 128.],
+                                                std=[1., 1., 1.])
+        pipe.set_outputs(cmn)
+    pipe.build()
+
```

Comment:
Maybe you can add a test that would go over all operators inside`l _cpu_ops`, `_gpu_ops`, `_mixed_opsand` and check the output from `GetArgumentNames`?

Reply:
I think it would be enough to just search for `output_dtype` in the SupportedOperators page. Unless you really want me to make sure it won't ever appear again?

Reply:
Fine